cypress/integration/cookieBanner.amp.js

```diff
@@ -0,0 +1,59 @@
+import config from '../support/config';
+
+const getPrivacyBanner = () =>
+  cy.contains("We've updated our Privacy and Cookies Policy");
+
+const getCookieBanner = () => cy.contains('Let us know you agree to cookies');
+
+describe('Amp Cookie Banner Test', () => {
+  beforeEach(() => {
+    cy.visit(`/news/articles/${config.assets.newsThreeSubheadlines}.amp`);
+  });
+
+  it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
+    getPrivacyBanner().should('be.visible');
+    getCookieBanner().should('not.be.visible');
+
+    cy.contains('OK').click();
+
+    getCookieBanner().should('be.visible');
+    getPrivacyBanner().should('not.be.visible');
+
+    cy.contains('Yes, I agree').click();
+
+    getCookieBanner().should('not.be.visible');
+    getPrivacyBanner().should('not.be.visible');
+  });
+
+  it('should show privacy banner if cookie banner isnt accepted, on reload', () => {
```

Comment:
_Is_ this the expected behaviour? I don't see it in #1400.

Reply:
Its not the desired behaviour, however we are limited to AMP I added
`Unlike the canonical banner, it doesnt dismiss the privacy banner once youve seen it. You have to click approve on the privacy AND cookie banner in the same page view for them to not return.` to https://github.com/bbc/simorgh/pull/1442 and its been tested to be the case.